### ISSUE TITLE
pronounce number with higher precision than provided

### ISSUE THREAD

**Issue by maxbachmann** [MycroftAI/mycroft-core#2397]
When using the function pronounce_number mycroft/util/lang/parse_*.py with a higher precision than the used number e.g.
```python
pronounce_number(-21.234, places=4)
```
in english the output is:
```
minus twenty one point two three four
```
while in german the output is
```
minus einundzwanzig komma zwei drei vier null
```

this leads to the question which of the two behaviors is correct, since either all pronounce_number functions should stop the sentence when they reach the precision of the provided number, or they should all add zero/null in the end until the requested precision is reached.

**Comment by ChanceNCounter** [comment]
What if it became, 
```
pronounce_number(number, places, trailing_zeroes=False)

**Comment by ChanceNCounter** [comment]
fixed by #2386 here and ported to Lingua Franca, I think this can be closed.

**Comment by maxbachmann** [comment]
This Commit did only fix some stuff with the way times are said, but did not work on the trailing zeros issue described in this issue

**Comment by ChanceNCounter** [comment]
Fair enough. In that case, could you (or a Mycroft person) please move it to LF? The function lives there now.

**Comment by maxbachmann** [comment]
This requires someone from Mycroft to move it (I could only close it here and open it again in LF)

**Comment by forslund** [comment]
I like to move it, move it...
